pkg/services/searchV2/bluge.go

```diff
@@ -528,9 +543,28 @@ func doSearchQuery(
 
 	locationItems := make(map[string]bool, 50)
 
+	docValuesFields := []string{
+		documentFieldUID,
+		documentFieldPanelType,
+		documentFieldKind,
+		documentFieldLocation,
+		documentFieldDSType,
+		documentFieldDSUID,
+		documentFieldTag,
+		documentFieldURL,
```

Comment:
URL -- I don't know if it makes a difference or not, but we could have it be stored but not searchable